javascript/frameworks/ui5/lib/advanced_security/javascript/frameworks/ui5/UI5XssQuery.qll

```diff
@@ -2,11 +2,14 @@ import javascript
 import advanced_security.javascript.frameworks.ui5.dataflow.DataFlow as UI5DataFlow
 import advanced_security.javascript.frameworks.ui5.UI5View
 import semmle.javascript.security.dataflow.DomBasedXssQuery as DomBasedXss
-import semmle.javascript.security.dataflow.ClientSideUrlRedirectCustomizations::ClientSideUrlRedirect as UrlRedirect
 
-class Configuration extends DomBasedXss::Configuration {
+class Configuration extends TaintTracking::Configuration {
```

Comment:
Similar to the query above. I don't think the main problem is that we extend `DomBasedXss::Configuration`. Instead, I think the duplication exists because we previously specified `UrlRedirect::LocationSink` as a sink. I think removing that line (as you have already done) will be sufficient here.